src/adstage/pluck_api.clj

```diff
@@ -33,7 +44,8 @@
                  (keyword? k) k
                  (map? k)     (-> k keys first))
                (cond
-                 (keyword? k) (-pluck k env init-result)
+                 (keyword? k) (or (k init-result)
```

Comment:
Worth considering - maybe use the default case instead of an or block here? Slightly different semantics.

```

;; init-result => {:foo nil :bar false :baz true}

(or (:foo init-result) (-pluck :foo env init-result)) ;; -> pluck is called
(or (:bar init-result) (-pluck :foo env init-result)) ;; -> pluck is called
(or (:baz init-result) (-pluck :foo env init-result)) ;; -> pluck is not called


(get init-result :foo (-pluck :foo env init-result)) ;; -> pluck is not called
(get init-result :bar (-pluck :foo env init-result)) ;; -> pluck is not called
(get init-result :baz (-pluck :foo env init-result)) ;; -> pluck is not called
```

This allows nil/false to be acceptable return types in the init result (I know datomic can't return nil, but it can probably return false, right?)